platform/xbox/functions/_PDCLIB/_PDCLIB_close.c

```diff
@@ -12,11 +12,18 @@
 #ifndef REGTEST
 
 #include "pdclib/_PDCLIB_glue.h"
-#include <hal/fileio.h>
+#include <windows.h>
 
-int _PDCLIB_close( int fd )
+int _PDCLIB_close( void *fd )
 {
-    return XCloseHandle( fd );
+    if ( CloseHandle( fd ) )
+    {
+        return 0;
+    }
+    else
+    {
+        return -1;
```

Comment:
A bit surprising that this isn't setting `errno` in some way

Reply:
That was an oversight, it does set it now.